src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -259,7 +259,11 @@
 
 @private commaSeparatedFields(args)
   @join arg : args on ", "
-    {@variableField(arg)}
+    @if arg.type.isEnum
```

Comment:
Can you check if this compiles? I *think* we need to import the `enums` package right?

Reply:
Added.